microdnf/context.cpp

```diff
@@ -310,6 +310,65 @@ void Context::load_rpm_repos(libdnf::repo::RepoQuery & repos, libdnf::rpm::Packa
 //     }
 // }
 
+
+void Context::download_and_run(libdnf::base::Transaction & transaction) {
```

Comment:
What about to move the logic to libdnf directly? Same code is also required by a daemon.

Reply:
Yeah I know. I'd rather leave it for the next iteration but if you want I can try to move the code.

Reply:
Ok, no problem